LDKNodeMonday/Service/KeyService/KeyService.swift

```diff
@@ -73,31 +71,25 @@ struct KeyClient {
 
     let saveNetwork: (String) throws -> Void
     let getNetwork: () throws -> String?
-    let saveEsploraURL: (String) throws -> Void
-    let getEsploraURL: () throws -> String?
-    let deleteNetwork: () throws -> Void
-    let deleteEsplora: () throws -> Void
```

Comment:
Did we move the `deleteNetwork` and `deleteEsplora` functionality to somewhere else or are they just not needed at all anymore?

Reply:
I removed them as they are not used anywhere anymore after removing the 'Full reset' option.
If we want to bring back the option to 'reset the node', I believe it is better to do that by deleting the documents folder containing the node on the network in question. 'Deleting' a network doesn't make that much sense when it is a required parameter of BackupInfo, saveNetwork, or the default on node start should cover the use cases.